### PR TITLE
chore(docs): auto-extend LVM thin pools configuration docs

### PR DESCRIPTION
**Pull request by prateekpandey14**
Signed-off-by: prateekpandey14 <prateekpandey14@gmail.com>

## Pull Request template

Please, go through these steps before you submit a PR.

**Why is this PR required? What issue does it fix?**:
Add docs and command to enable the monitoring on thin pools to use the auto extend feature of lvm volumes.

**Checklist:**
- [ ] Fixes #<issue number>
- [x] PR Title follows the convention of  `<type>(<scope>): <subject>`
- [ ] Has the change log section been updated?
- [ ] Commit has unit tests
- [ ] Commit has integration tests
- [ ] (Optional) Are upgrade changes included in this PR? If not, mention the issue/PR to track:
- [ ] (Optional) If documentation changes are required, which issue on https://github.com/openebs/openebs-docs is used to track them:

### CODE REVIEW

File: docs/thin_provision.md [hunk 1]
```diff
@@ -49,7 +49,36 @@ $ lvextend -L +15G lvmvg/thin_pool
...
+  LV             VG    Attr       LSize Pool           Origin Data%  Meta%  Move Log Cpy%Sync Convert Monitor
+  lvmvg_thinpool lvmvg twi-aotz-- 2.07g                       0.00   11.52                            not monitored
+```
+
+If the output in the Monitor column reports, as above, that the volume is not monitored,
+then monitoring needs to be explicitly enabled. Without this step, automatic extension
+of the logical thin pool will not occur, regardless of any settings in the `lvm.conf`.
+
+```
+$ lvchange --monitor y lvmvg/lvmvg_thinpool
```
Comment (akhilerm) [inline docs/thin_provision.md:69]: @prateekpandey14 Is this something that the user has to do manually?, or will it be done as part of the first thin pool creation?

Comment (prateekpandey14) [reply docs/thin_provision.md:69]: User has to run manually

File: docs/thin_provision.md [hunk 2]
```diff
@@ -49,7 +49,36 @@ $ lvextend -L +15G lvmvg/thin_pool
 
 ### Configure Auto Extending of the Thin Pool (Configure Over-Provisioning protection)
 
-Editing the settings in the `/etc/lvm/lvm.conf` can allow auto growth of the thin 
+1. Since we already create thin pool as part of first thin volume provisioning,
```
Comment (pawanpraka1) [inline docs/thin_provision.md:52]: change this :
we already create thin pool  => we automatically create thin pool

**Review by pawanpraka1 — APPROVED** [review]
can we create one issue to improve the user experience.

File: docs/thin_provision.md (hunk 2 above)
Comment (prateekpandey14) [reply docs/thin_provision.md]: done

**Comment by prateekpandey14** [comment]
>can we create one issue to improve the user experience.

Created to track it further #93

**Pull request merged**